content/1.7/back-office/dashboard/dashboard-configure.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Dashboard Configure
+weight: 1
+---
+
+# Dashboard configure
+
+3 blocks are displayed on this page:
+
+ - **Average bank fees per payment method**
+ - **Average shipping fees per shipping method**
+ - **Other settings**
+
+## Demo mode
+
+ - **Demo mode**: This toggle allows to enable or disable fake data. By default, it is disabled. When enabled, fake data are used. When disabled, the real store's data are used. The store's data are gathered by PrestaShop's statistics system.
+The following tooltip is displayed: "This mode displays sample data so you can try your dashboard without real numbers."
+
+## Average bank fees per payment method
+
+4 Input fields are presented in this block, these are the default fields, if more payment methods are added to the shop, they will be displayed additionally here:
```

Comment:
"if more payment methods are added to the shop, they will be displayed additionally here": I am not sure to understand this sentence. You mean if others payments modules are installed ?

On my side I have only 2 payments modules installed (Bank transfer and Payments by check). However, I have 4 fields for bank transfer and 4 for Payments by check

Reply:
@marionf I wasn't sure if I should add this, but if merchant has for example any other modules installed or new payment methods added, they would be shown here, I hope that makes sense now